samples/matter/window_covering/boards/nrf54l15pdk_nrf54l15_cpuapp_release.conf

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Multirole is the only currently supported role by SoftDevice.
+CONFIG_BT_LL_SOFTDEVICE_MULTIROLE=y
+
+# TODO: Workaround to be removed once DFU will be supported on nRF54L.
+CONFIG_CHIP_OTA_REQUESTOR=n
+CONFIG_CHIP_QSPI_NOR=n
+
+CONFIG_FPROTECT=n
+CONFIG_LOG_MODE_DEFERRED=y
```

Comment:
Should it be disabled `CONFIG_LOG` in release?

Reply:
Good catch, thanks! :)